channeldb/migrations.go

```diff
@@ -474,20 +473,24 @@ func migratePruneEdgeUpdateIndex(tx *bolt.Tx) error {
 	if edges == nil {
 		return nil
 	}
-	edgeUpdateIndex := edges.Bucket(edgeUpdateIndexBucket)
-	if edgeUpdateIndex == nil {
-		return nil
+	edgeUpdateIndex, err := edges.CreateBucketIfNotExists(
+		edgeUpdateIndexBucket,
+	)
+	if err != nil {
+		return fmt.Errorf("unable to create/fetch edge update " +
+			"index bucket")
```

Comment:
Not sure how helpful the errors returned by `CreateBucketIfNotExists` are, but we could include them here.

Reply:
IIRC, it doesn't give any more context than this.